src/App.js

```diff
@@ -90,22 +71,48 @@ function App() {
     />
   );
 
+  const handlePositionClick = () => {
+    allowMarker(true);
+    const options = {
+      enableHighAccuracy: true,
+      timeout: 5000,
+      maximumAge: 0,
+    };
+    function success(pos) {
```

Comment:
I'm a fan of this pattern of encapsulating functions that are only used within the scope of the handler inside it like this. Nice work!